test/utils/DateUtils-test.ts

```diff
@@ -44,12 +44,12 @@ describe("formatRelativeTime", () => {
     });
 
     it("returns hour format for events created less than 24 hours ago", () => {
-        const date = new Date(1635850883000);
+        const date = new Date(2021, 10, 2, 11, 1, 23, 0);
         expect(formatRelativeTime(date)).toBe("11:01");
     });
 
     it("honours the hour format setting", () => {
-        const date = new Date(1635850883000);
+        const date = new Date(2021, 10, 2, 11, 1, 23, 0);
```

Comment:
ftr: months are zero indexed, because why not. 

![image](https://user-images.githubusercontent.com/1190097/142948872-381e2bec-8c72-4e1f-82b0-413e6fd357da.png)